awt/IKVM.AWT.WinForms.FirstPass/IKVM.AWT.WinForms.FirstPass.csproj

```diff
@@ -1,11 +1,11 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFrameworks>net461;netstandard2.0</TargetFrameworks>
-    <OutputPath>..\..\bin\$(Configuration)</OutputPath>
+    <TargetFrameworks>net461;netcoreapp3.1</TargetFrameworks>
```

Comment:
I thought we wanted .NET Standard not .NET Core?

Reply:
We can't do netstandard without significant rework of IKVM.  There are many incompatibilities, like things exist in Framework only or in Core only.  And IKVM's design is quite Framework specific.
So, a decision was made to have two versions, Framework and CoreApp as CoreApp required a least amount of changes.

Reply:
@mbinsfeld That was my call - standard would be an immense amount of additional work.

Reply:
@igorkwwrd, can you elaborate on what may be involved for .NET Standard 2.x support? What specifically requires lots of rework?